src/server/index.test.jsx

```diff
@@ -952,13 +952,6 @@ describe('Server', () => {
   });
 
   describe('Service workers', () => {
-    it('should serve a file for existing service workers', async () => {
-      await makeRequest('/news/articles/sw.js');
-      expect(sendFileSpy.mock.calls[0][0]).toEqual(
-        path.join(__dirname, '/public/sw.js'),
-      );
-    });
-
     it('should not serve a file for non-existing service workers', async () => {
```

Comment:
Could we update this test to use one of the public services as the test (instead of `some-service`)